### PR TITLE
site: add ant-design-vue link

### PR DESCRIPTION
**Pull request by tangjinzhou**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### Add more navigation to ant-design-vue.

### CODE REVIEW

File: site/theme/template/Home/Page2.jsx [hunk 1]
```diff
@@ -79,6 +79,9 @@ export default function Page2({ isMobile, locale }) {
       <a href="https://ng.ant.design/" target="_black">
         Ant Design of Angular <Icon type="right" />
       </a>
+      <a href="https://vuecomponent.github.io/ant-design-vue" target="_black">
+        Ant Design of Vue <Icon type="right" />
+      </a>
```
Comment (afc163) [inline site/theme/template/Home/Page2.jsx]: 如果希望成为官方推荐，有下面几点希望可以做到。

- [ ] 持续的团队维护，Ant Design 版本迭代跟进。
- [ ] 弱化 README.md 里的赞助链接部分，特别是支付宝和微信二维码，现在过于显眼了。
- [ ] 修复一些明显的站点样式错误：https://github.com/vueComponent/ant-design-vue/issues/117

Comment (afc163) [reply site/theme/template/Home/Page2.jsx]: 和 ng-zorro 对比，如果后面做得特别好的话（三个月内保持活跃迭代之类的），域名可以直接上 vue.ant.design，然后我找设计师设计一个好看一点的 logo。

Comment (tangjinzhou) [reply site/theme/template/Home/Page2.jsx]: 明白，我们会努力。已经在做v3.8的迭代跟进。

**Comment by afc163** [comment]
Wait for new logo

**Pull request closed without merging**